Missing #include <string> in src/gui/dialogs/browser/browserBase.h

**Environment**
- OS: Fedora Rawhide (35)
- Giada version: 0.17.2

**Describe the bug**
Fails to compile with errors like:

```
In file included from ../src/gui/dialogs/browser/browserDir.h:30,
                 from ../src/gui/dialogs/browser/browserDir.cpp:27:
../src/gui/dialogs/browser/browserBase.h:57:14: error: 'string' in namespace 'std' does not name a type
   57 |         std::string getSelectedItem() const;
      |              ^~~~~~
```

**To Reproduce**
Steps to reproduce the behavior:
1. Use a Fedora 35/Rawhide VM (or something with the same g++ and standard library versions).
2. Compile normally.

Or just look at the source to see the missing include.

**Expected behavior**
No compiler errors.

**Screenshots**
N/A

**Additional context**
The file `src/gui/dialogs/browser/browserBase.h` uses `std::string`, so it should `#include <string>` directly rather than relying on an indirect include via some other header. I have confirmed that this fixes the problem.

--- a/src/gui/dialogs/browser/browserBase.h
+++ b/src/gui/dialogs/browser/browserBase.h
@@ -30,6 +30,7 @@
 #include "core/types.h"
 #include "gui/dialogs/window.h"
 #include <functional>
+#include <string>
 
 class Fl_Group;
 class geCheck;

